index.css

```diff
@@ -203,16 +221,15 @@ body {
 }
 
 .underline {
-  border: 1px solid #212121;
-  flex: none;
-  order: 1;
-  flex-grow: 0;
-  margin: 24px 5% 80px 5%;
+  border-top: 1px solid #212121;
+  width: 98%;
+  align-self: center;
+  margin-left: 2%;
 }
 
 .recent_works {
   display: flex;
-  flex-direction: column;
+  flex-direction: row;
```

Comment:
- [ ] Kindly adjust the spacing between your img and text to look as close as possible to your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206721100-491930c0-c2fd-47a2-b94c-a8ae42b3655b.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206721065-9d19ebf8-1805-45ee-84f3-070fcf4305d3.png)